index.html

```diff
@@ -11,12 +11,12 @@
 
     <section>
       <nav class="left-nav">
```

Comment:
Great addition, future feauture could be to make the images take you to "google search images" of that query input

I like the idea of putting it into the results and hiding the Searchbar/logo; Also, as you mentioned, make the searchbar and logo at the top once search is clicked/ or redirecting to a new page using "window.location.href = "./searchResults.html"; in the j.s. function -->

l